response.go

```diff
@@ -93,7 +93,7 @@ func (r *Response) Cookies() []*http.Cookie {
 
 // String method returns the body of the server response as String.
 func (r *Response) String() string {
-	if r.body == nil {
+	if len(r.body) == 0 {
```

Comment:
@haivp3010 Can you update this line as -
```go
if r.body == nil  || len(r.body) == 0 {
```

Reply:
@jeevatkm I think it makes no difference here. Whether `r.body` is `nil` or empty, calling `len` on it would still return 0.

https://pkg.go.dev/builtin#len

```
The len built-in function returns the length of v, according to its type:

Array: the number of elements in v. Pointer to array: the number of elements in *v (even if v is nil). Slice, or map: the number of elements in v; if v is nil, len(v) is zero. String: the number of bytes in v. Channel: the number of elements queued (unread) in the channel buffer; if v is nil, len(v) is zero.
```